lib/models/activity-log.js

```diff
@@ -0,0 +1,22 @@
+const { Model } = require('objection');
+
+class ActivityLog extends Model {
+  static get tableName() {
+    return 'activity_log';
+  }
+
+  static get relationMappings() {
+    return {
+      case: {
+        relation: this.BelongsToOneRelation,
+        modelClass: `${__dirname}/../db/case`,
```

Comment:
Seeing this reminds me that the `db` directory is the correct place for this file to live. Since that's where the other _database_ models live.